frontend/public/index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
```

Comment:
A Bootstrap 5-ös verziója még mindig csak béta állapotban van, én lehet a 4.6-ot húznám be. Ha marad az 5 akkor figyeljetek, hogy ha kerestek valamire akkor régi válaszok lehetnek félrevezetőek. (nem tudom pontosan mi változik az ötös verzióban)